app/src/main/java/lt/getpet/getpet/fragments/UserProfileFragment.kt

```diff
@@ -47,6 +48,10 @@ class UserProfileFragment : BaseFragment(), AuthStateChangedListener {
         button_login.setOnClickListener {
             navigationManager.navigateToUserLoginActivity(activity!!)
         }
+
+        button_signout.setOnClickListener {
+            onUserSignedOut(activity!!)
```

Comment:
Don't pass `activity` instead of `context`. `Context` is already provided by fragment.